python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ptp.j2

```diff
@@ -9,21 +9,6 @@
 {%     if ptp.clock_identity is arista.avd.defined %}
 ptp clock-identity {{ ptp.clock_identity }}
 {%     endif %}
-{%     if ptp.source.ip is arista.avd.defined %}
-ptp source ip {{ ptp.source.ip }}
-{%     endif %}
-{%     if ptp.priority1 is arista.avd.defined %}
-ptp priority1 {{ ptp.priority1 }}
-{%     endif %}
-{%     if ptp.priority2 is arista.avd.defined %}
-ptp priority2 {{ ptp.priority2 }}
-{%     endif %}
-{%     if ptp.profile is arista.avd.defined %}
-ptp profile {{ ptp.profile }}
-{%     endif %}
-{%     if ptp.ttl is arista.avd.defined %}
-ptp ttl {{ ptp.ttl }}
-{%     endif %}
```

Comment:
ptp message-type event will come before to ptp message-type general

Reply:
Fixed